skpro/distributions/qpd.py

```diff
@@ -0,0 +1,705 @@
+"""Johnson Quantile-Parameterized Distributions."""
+# copyright: skpro developers, BSD-3-Clause License (see LICENSE file)
+
+__author__ = [
+    "FelixWick",
+    "setoguchi-naoki",
+]  # interface only. Cyclic boosting authors in cyclic_boosting package
+
+import warnings
+from typing import Optional
+
+import numpy as np
+import pandas as pd
+from cyclic_boosting.quantile_matching import (
```

Comment:
these imports should move into the distribution classes.

My recommendation is to move them inside the methods where they are needed.

Reply:
done in my commits